packages/mobile/src/screens/track-screen/RemixContestCountdown.tsx

```diff
@@ -0,0 +1,73 @@
+import { useRemixContest } from '@audius/common/api'
+import { useRemixCountdown } from '@audius/common/hooks'
+import type { ID } from '@audius/common/models'
+import { formatDoubleDigit } from '@audius/common/utils'
+import { css } from '@emotion/native'
+
+import { Text, Flex, Divider } from '@audius/harmony-native'
+
+const messages = {
+  days: 'days',
+  hours: 'hours',
+  minutes: 'minutes',
+  seconds: 'seconds'
+}
+
+type TimeUnitProps = {
+  value: number
+  label: string
+}
+
+const TimeUnit = ({ value, label }: TimeUnitProps) => {
+  return (
+    <Flex column gap='2xs' alignItems='center' justifyContent='center'>
+      <Text
+        variant='label'
+        color='inverse'
+        style={css({ fontSize: 24, lineHeight: 24 })}
```

Comment:
seems like we don't have size='2xl' for label variant :(

Reply:
and in designs they are not referred to as such, just have custom height/lineheight

Reply:
the closest mapping is heading regular?

Reply:
omg it's heading! tysm the designs didn't say this